R/tabyl.R

```diff
@@ -247,7 +247,7 @@ tabyl_3way <- function(dat, var1, var2, var3, show_na = TRUE, show_missing_level
   # grab class of 1st variable to restore it later
   col1_class <- class(dat[[1]])
   col1_levels <- NULL
-  if ("factor" %in% col1_class) {
+  if (is.factor(dat[[1]])) {
```

Comment:
For this change, I tried to see if there was a specific reason why is.factor was not used. but I couldn't find any notes.

```r
is.factor(ordered(c("1", "2")) 
#> TRUE
```

This line seems to have been implemented in #176, but no note on why is.factor was not used. and was fixed in #386 to what it is today.